build-tools/automation/azure-pipelines.yaml

```diff
@@ -1081,7 +1081,8 @@ stages:
   - job: designer_integration_win
     condition: false
     displayName: Windows
-    pool: $(HostedWinVS2019)
```

Comment:
Not sure I understand this change.  Shouldn't we *want* to use a variable?

Reply:
I didn't add a variable for this since it's now the only job using the 2019 pool. I also think we will want to update it to use the 2022 image defined in `$(HostedWinImage)` when we re-enable it.